ansible/roles/package-upgrade/tasks/partials/yum.yml

```diff
@@ -6,3 +6,5 @@
 
   - name: upgrade installed packages
     yum: name=* state=latest
+# XXX might have needed `use_backend=yum` at end of last line, but prob
```

Comment:
we can remove this comment before landing

Reply:
I wonder if this could break other platforms that need a `dnf`/`yum4` backend. We may need to condition this by os or set an `ansible_pkg_mgr` fact
https://docs.ansible.com/ansible/latest/modules/yum_module.html

Reply:
If this is about getting ansible to recognise `yum` on IBM i I have a one-line patch for ansible to add `/QOpenSys/pkgs/bin/yum` to the [list of `PKG_MGRS`](https://github.com/ansible/ansible/blob/cdaf7da11a2cdffe7c9bd5cff7d1b2acfa8e95e1/lib/ansible/module_utils/facts/system/pkg_mgr.py#L16-L40):
```diff
--- /home/users/riclau/.local/lib/python2.7/site-packages/ansible/module_utils/facts/system/pkg_mgr.py.old      2020-04-23 12:22:58.779566067 -0400
+++ /home/users/riclau/.local/lib/python2.7/site-packages/ansible/module_utils/facts/system/pkg_mgr.py  2020-04-23 12:24:16.201566043 -0400
@@ -37,6 +37,7 @@
             {'path': '/usr/sbin/sorcery', 'name': 'sorcery'},
             {'path': '/usr/bin/rpm-ostree', 'name': 'atomic_container'},
             {'path': '/usr/bin/installp', 'name': 'installp'},
+            {'path': '/QOpenSys/pkgs/bin/yum', 'name': 'yum'},
             ]
```

I'll look at submitting this to ansible.

Reply:
https://github.com/ansible/ansible/pull/69484

Reply:
FWIW it looks like https://github.com/ansible/ansible/pull/69484 went out in ansible v2.10.0: https://github.com/ansible/ansible/blob/v2.10.1/changelogs/CHANGELOG-v2.10.rst#v2-10-0